pavelib/servers.py

```diff
@@ -27,10 +31,12 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 
     if not settings:
         settings = DEFAULT_SETTINGS
+        asset_settings = settings
```

Comment:
Good point. The problem was that some callers assumed that they didn't have to specify any settings and the defaults would work, but then the assets wouldn't get updated. I'll try removing this line and see what else I have to fix up to make it consistent.